container_workflow_tool/distgit.py

```diff
@@ -276,6 +307,9 @@ def dist_git_changes(self, images, rebase=False):
                     # Clone upstream repository
                     ups_path = os.path.join('upstreams/', ups_name)
                     self._clone_upstream(url, ups_path, commands=commands)
+                    test_openshift_yaml_file = os.path.join(ups_path, "test", "test-openshift.yaml")
+                    if os.path.exists(test_openshift_yaml_file):
+                        self.update_openshift_yaml(test_openshift_yaml_file, version=path)
```

Comment:
It seems to me, that this update_openshift_yaml function definition is missing in distgit.py